src/System.Text.Encoding.CodePages/tests/EncoderFallbackBufferHelper.cs

```diff
@@ -0,0 +1,91 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Text.Tests
+{
+    public class TestEncoderFallbackBuffer : EncoderFallbackBuffer
```

Comment:
>TestEncoderFallbackBuffer [](start = 17, length = 25)

do we need to write other one for the decoder too?

Reply:
Yes, I'll add in a separate PR.